Update rules engine (#501)
akeneo/pim-helpcenter: content/md/first-steps/manage-your-rules.md

---
id: manage-your-rules
themes: boost-your-productivity, catalog-settings
title: Manage your **rules**
popular: false
ee-only: true
related: what-is-a-rule
---

# Overview

A rule allows you to automatically enrich some information in your products, according to automatic actions based on conditions. It applies on attributes, categories, system attributes (status, completeness...).  

> For more details, please refer to [the dedicated article](blabla.html).

Reviewer: Does the page "blabla.html" exists?

Author: Nope, I'll add the link afterward. This is the second article that should be soon ready ;)

# View all rules

> The `Settings/Rules` screen displays all the rules that have been created and then imported in the PIM.  

Reviewer 2: create and import rules are the same action (for now)

# Create a rule

> Rules are exclusively created by YML imports. You need to create your YML file like explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.  

Reviewer 3: ```suggestion
You can only create rules by YML  imports. You need to create your YML file as explained on our online technical documentation: [General information about rule format](https://docs.akeneo.com/latest/manipulate_pim_data/rule/general_information_on_rule_format.html#enrichment-rule-structure) and import it in the PIM using the `Rule import in YML` import job.  
```

> We also detail in [this article](get-started-with-the-rules-engine.html), how to select products and the list of available actions.

Reviewer 3: ```suggestion
[Here](get-started-with-the-rules-engine.html) we also explain how to select products and the list of available actions.
```

::: info

> For more details about how to execute an import, please refer to [Import your data](imports.html).

Reviewer 3: ```suggestion
For more details on how to execute an import, please refer to [Import your data](imports.html).
```

:::

> You can also export your rules. For that, execute the `Rule export in YML` export job.

Reviewer 3: ```suggestion
You can also export your rules. To do so, you need to execute the `Rule export in YML` export job.
```

::: info

> For more details about how to execute an export, please refer to [Export your data](exports.html).

Reviewer 3: ```suggestion
For more details on how to execute an export, please refer to [Export your data](exports.html).
```

> You can also delete several rules at the same time:

Reviewer 3: ```suggestion
You can also delete several rules at once:
```

1.  Select one or several rules by ticking them on the left side of the screen  
1.  Click on `Delete`

> 1.  Then confirm the action in the pop in.

Reviewer 3: ```suggestion
1.  Then, confirm the action in the pop in.
```
